PROCESS.md

```diff
@@ -77,6 +77,24 @@ Once the above process has taken its course, make sure you
 
 **Applying as core maintainer:** The same process applies for applying as a core maintainer. The PR should be against the [CORE-MAINTAINERS file](https://github.com/fluxcd/community/blob/main/CORE-MAINTAINERS) though and an accordingly higher level of experience and a more holistic understanding of the project is expected.
 
+### Applying as a Security Team member
+
+**Process:**
+
+- You need to be a core maintainer for applying
```

Comment:
I do not think it is a strict requirement to be part of the Core Maintainer team. E.g. Scott who currently holds a position isn't, but is involved with all meta things around Flux.

In addition, having at least one member who isn't a Core Maintainer ensures that if all hell breaks loose around "core issues", there will still be a person with less on their plate to prepare and/or handle the communication around security issues.

Reply:
Ok, that works for me. I will change the language accordingly.